firebase-firestore/src/main/java/com/google/firebase/firestore/core/Bound.java

```diff
@@ -70,14 +71,22 @@ public String canonicalString() {
     return builder.toString();
   }
 
+  public String getFieldValue() {
```

Comment:
Per my comment on getStartAt() / getEndAt(), this should probably go away (we don't want / need the field value as a string).